pkg/operator/apiserver/audit/audit_policies_test.go

```diff
@@ -38,37 +38,42 @@ func TestDefaultPolicy(t *testing.T) {
 
 func TestGetAuditPolicy(t *testing.T) {
 	scenarios := []struct {
-		name       string
-		goldenFile string
-		config     configv1.Audit
+		name        string
+		goldenFile  string
+		config      configv1.Audit
+		errContains string
 	}{
 		{
 			name: "Default",
 			config: configv1.Audit{
 				Profile: "Default",
 			},
-			goldenFile: "default.yaml",
+			goldenFile:  "default.yaml",
+			errContains: "",
```

Comment:
nit: you can drop these empty fields where no error is expected

Reply:
I think this is controlling that the Len is 0, and if dropped it may not work

Reply:
if you drop it, it is the zero value which is `""`.

Reply:
This worked. Guess it's implied value.

Reply:
Resolved